docs/developers/building-contributing.md

```diff
@@ -16,7 +16,7 @@
 ## Testing
 Normal test invocation is just `make test`. Run a single test with an invocation like `go test -v -run TestDevAddSites ./pkg/...`
 
-If you set the environment variable DRUD_DEBUG=true you can see what ddev commands are being executed in the tests.
+To see which ddev commands the tests are executing, set the environment variable to DRUD_DEBUG=true.
```

Comment:
"set the DRUD_DEBUG environment variable to TRUE", or "set the environment variable DRUD_DEBUG=true"

Reply:
oops! good catch. fixing!